Dockerfile

```diff
@@ -7,25 +7,17 @@ COPY . .
 
 # Update and install needed deps prioir to installing the binary.
 RUN apk update && \
-    apk --no-cache add make=4.2.1-r2 git=2.24.3-r0 && \
+    apk --no-cache add make git && \
     make install
 
-FROM alpine:edge
+FROM alpine:latest
 
 ENV RELAYER /relayer
 
-RUN addgroup rlyuser && \
```

Comment:
Looks fine, but why use the sudo user?

Reply:
I concur, could we add it back? Using users aligns with gaia dockerfile and anyone using a relayer in production should know how to run the relayer in a docker container as a regular user